backend/models/core/Entity.js

```diff
@@ -122,6 +122,90 @@ class Entity {
         })); 
   }
 
+  static async findByPageAndEntry({entity_type_slug , entry, page }) {
+    const db = new DB();
+     
+    let totalRows;
+    let totalOrders;
+    
+    const totalRowsSQL = `select COUNT(entities.id) 
```

Comment:
Capitalize all predefined SQL statements (e.g. SELECT, INSERT, UPDATE etc)

```suggestion
    const totalRowsSQL = `SELECT COUNT(entities.id) 
```